### ISSUE TITLE
[BUG] Unauthorized error caused by invalid NTP config/timestamp difference

### ISSUE THREAD

**Issue by TachunLin** [harvester/harvester#2119]
**Describe the bug**

While one click upgrading a 3 nodes harvester from v1.0.0 to v1.0.1-rc2.
The upgrade process stuck in the `Upgrading System Servide` showing error message:
**`Job has reached the specified backoff limit`**

![image](https://user-images.githubusercontent.com/29251855/161921040-460c3670-6206-4e5a-952c-917af3c2a683.png)

**To Reproduce**
1. Prepare v1.0.0 three nodes harvester cluster on Provo machines
2. Create a ubuntu and SLES JeOS image
3. Create a ubuntu and SLES virtual machine
4. Enable network to `harvester-mgmt`
5. Create a virtual network `vlan1` with id `1`
6. Setup back target to AWS S3
7. Create two backups for the ubuntu and SLES VM
8. Shutdown all VMs
9. Follow the steps to upgrade to v1.0.1-rc2
https://github.com/harvester/docs/blob/42598035cfe79856f7e159d4c47c4d7d29178900/docs/upgrade/automatic.md

**Expected behavior**

Should complete the `upgrade system service` process and successfully finish all one click upgrade process

**Support bundle**

[supportbundle_2c72269f-cc63-4361-afb9-2e128771492d_2022-04-01T10-47-02Z.zip](https://github.com/harvester/harvester/files/8424673/supportbundle_2c72269f-cc63-4361-afb9-2e128771492d_2022-04-01T10-47-02Z.zip)

**Environment:**
 - Harvester ISO version: v1.0.0 GA to v1.0.1-rc2
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): 3 nodes bare machines on Provo

**Additional Context**
![image](https://user-images.githubusercontent.com/29251855/161922317-ebf4998e-e057-4ea7-9f87-2094e45f4be3.png)

**Comment by bk201** [comment]
Could be related to the time difference between nodes.
```
Warning: Permanently added '10.84.44.114' (ED25519) to the list of known hosts.
Wed Apr  6 04:11:09 UTC 2022
Warning: Permanently added '10.84.44.115' (ED25519) to the list of known hosts.
Wed Apr  6 04:13:46 UTC 2022
Warning: Permanently added '10.84.44.116' (ED25519) to the list of known hosts.
Wed Apr  6 04:12:56 UTC 2022
```

**Comment by bk201** [comment]
I can reproduce this with VMs by manually setting time difference between nodes, here is what I saw these logs in `kube-apiserver-node1` pod:
```
E0406 10:49:21.991249       1 claims.go:126] unexpected validation error: *errors.errorString
E0406 10:49:21.991291       1 authentication.go:63] "Unable to authenticate the request" err="[invalid bearer token, Token could not be validated.]"
```

**Comment by bk201** [comment]
I had added [a warning paragraph in the upgrade section](https://github.com/harvester/docs/pull/131/files#diff-9b99ca129b131cc50e1c584be79715640745a08c667b20f0fa714558b2d3077aR29-R45) to describe the need of NTP server.

@FrankYang0529 Could you help update the rest of the doc? Here are some places that need to improve:
- https://docs.harvesterhci.io/v1.0/install/iso-install/ 
  - `(Optional) Configure the NTP Servers of the node. This defaults to 0.suse.pool.ntp.org.` Probably change to recommended.
- https://docs.harvesterhci.io/v1.0/install/pxe-boot-install/ 
  - Update the example to include NTP servers.
- https://docs.harvesterhci.io/v1.0/install/harvester-configuration/
  - Update the example to include NTP servers.
- https://docs.harvesterhci.io/v1.0/install/harvester-configuration/#osntp_servers
  - Highly recommend the option here again.

**Comment by guangbochen** [comment]
And the https://github.com/harvester/ipxe-examples if we can adjust the examples with some default NTP servers, thanks.

**Comment by TachunLin** [comment]
Verified **fixed** on **`v1.0.1-rc3`**. Close this issue.

### Result
After we add specific ntp server to each node for the config-create and config-join harvester configuration file for PXE install. 

  ```
  os:
    hostname: hpd8s7      
    ssh_authorized_keys:
    - ssh-rsa ...
    ntp_servers:
      - 0.pool.ntp.org
  ```

And check all nodes have identical time alignment with ntp service. 
During the upgrade from v1.0.0 to v1.0.1-rc3, we did not encounter the issue of stuck in  **`Job has reached the specified backoff limit`**

![image](https://user-images.githubusercontent.com/29251855/162426871-f51a86dc-c9a6-49f8-8d47-e07e2eacbe1c.png)


### Test Information
* Test Environment: 3 nodes harvester on provo bare machine
* Harvester version: v1.0.0 to v1.0.1-rc3

### Verify Steps
1. Add the ntp server to config-create and config-join yaml file
  ```
  ntp_servers:
      - 0.pool.ntp.org
  ```
2. Create a 3 nodes harvester with v1.0.0 release 
3. SSH to each node 
4. Run `timedatectl show-timesync` to check the ntp server and date time are aligned between each nodes
5. Upgrade harvester to v1.0.1-rc2 according to the steps https://github.com/harvester/harvester/issues/2119#issue-1194199084

### Additional Context
* Support Bundle
[supportbundle_d378036d-5fc2-4f82-938a-4f8602a01d3b_2022-04-08T11-31-45Z.zip](https://github.com/harvester/harvester/files/8451421/supportbundle_d378036d-5fc2-4f82-938a-4f8602a01d3b_2022-04-08T11-31-45Z.zip)
